Tests/OpenCombineTests/PublisherTests/MapTests.swift

```diff
@@ -0,0 +1,156 @@
+//
+//  MapTests.swift
+//
+//
+//  Created by Anton Nazarov on 25/06/2019.
+//
+
+import XCTest
+
+#if OPENCOMBINE_COMPATIBILITY_TEST
+import Combine
+#else
+import OpenCombine
+#endif
+
+@available(macOS 10.15, *)
+final class MapTests: XCTestCase {
+    static let allTests = [
+        ("testEmpty", testEmpty),
+        ("testError", testError),
+        ("testRange", testRange),
+        ("testDemandSubscribe", testDemandSubscribe),
+        ("testDemandSend", testDemandSend),
+        ("testCompletion", testCompletion),
+        ("testCancel", testCancel),
+        ("testCancelAlreadyCancelled", testCancelAlreadyCancelled)
+    ]
+
+    func testEmpty() {
+        // Given
+        let tracking = TrackingSubscriberBase<Never>(receiveSubscription: { $0.request(.unlimited) })
+        let publisher = PassthroughSubject<Int, Never>()
+        // When
+        publisher.map { $0 * 2 }.subscribe(tracking)
+        // Then
+        XCTAssertEqual(tracking.subscriptions.count, 1)
```

Comment:
This assert is redundant, as `tracking.subscriptions` is just a lazy view on `tracking.history`, which is already checked by the assert below.

Reply:
Deleted